index.html

```diff
@@ -41,6 +41,191 @@ <h1 class="primary-heading"> I’m Mariane Glad to see you! </h1>
 
         </ul>
     </section>
+
+    <section class="Projects">
+        <section class="Card-Works">
+            <header class="Snapshot">
+                <img src="./img/Snapshoot Portfolio.png" alt="snapshot poortfolio" class="snapshot">
+
+            </header>
+            <div class="info-block">
+                <h1 class="project-title">Tonic</h1>
+                <ul class="frame">
+                    <span class="client">
+                        CANOPY</span>
+                    <img src="./img/Counter.png" alt="" class="counter">
+                    <span class="role">Back End Dev</span>
+                    <img src="./img/Counter.png" alt="" class="counter">
+                    <span class="year">2015</span>
+    
+                </ul>
+                <span class="details">
```

Comment:
I suggest you use a `<p>` tag for this paragraph here. Using the right html element ensures screen readers are not confused when they parse your site.